js/foam/grammars/CSSRewriteURL.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+CLASS({
+  package: 'foam.grammars',
+  name: 'CSSRewriteURL',
+
+  requires: [
+    'XHR',
+    'foam.grammars.CSSDecl',
+    'foam.util.Base64Encoder',
+  ],
+  imports: [
+    'assert',
+    'document',
+  ],
+
+  documentation: function() {/* A slightly modified
+    $$DOC{ref:'foam.grammars.CSSDecl'} prefix-rewrite parser that also knows how
+    to expand remote URLs to dataURLs after fetching the data.
+  */},
+
+  constants: {
+    PREFIXES: [
+      '-webkit-'
+    ],
+    PREFIXED_KEYS: {
```

Comment:
They shouldn't be. Fixed in latest diff.

On 25 November 2015 at 15:14, Kevin Glen Roy Greer <notifications@github.com
> wrote:

> In js/foam/grammars/CSSRewriteURL.js
> <https://github.com/foam-framework/foam/pull/476#discussion_r45914330>:
>
> > +  ],
> > +  imports: [
> > +    'assert',
> > +    'document',
> > +  ],
> > +
> > +  documentation: function() {/* A slightly modified
> > +    $$DOC{ref:'foam.grammars.CSSDecl'} prefix-rewrite parser that also knows how
> > +    to expand remote URLs to dataURLs after fetching the data.
> > +  */},
> > +
> > +  constants: {
> > +    PREFIXES: [
> > +      '-webkit-'
> > +    ],
> > +    PREFIXED_KEYS: {
>
> Why are these duplicated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/foam-framework/foam/pull/476/files#r45914330>.
>